ciao-image/datastore/metads.go

```diff
@@ -27,39 +28,51 @@ type MetaDs struct {
 
 // Write is the metadata write implementation.
 func (m *MetaDs) Write(i Image) error {
-	err := m.DbAdd("images", i.ID, &i)
+	tenant := i.TenantID
+	if i.Visibility == imageAPI.Public {
+		tenant = string(imageAPI.Public)
```

Comment:
I added an alias ``imageAPI`` for image package. But, yeah, there's no need of that alias. /me changed it.